components/omega/src/ocn/TendencyTerms.cpp

```diff
@@ -43,6 +43,13 @@ VelocityHyperDiffOnEdge::VelocityHyperDiffOnEdge(const HorzMesh *Mesh)
       DcEdge(Mesh->DcEdge), DvEdge(Mesh->DvEdge),
       MeshScalingDel4(Mesh->MeshScalingDel4), EdgeMask(Mesh->EdgeMask) {}
 
+WindForcingOnEdge::WindForcingOnEdge(const HorzMesh *Mesh)
+    : SaltWaterDensity(1.026e3) {}
```

Comment:
Is this meant to be the reference ocean density? If so, I think it should come from the namelist.

Reply:
I changed this variable to come from the namelist
```yaml
Tendencies:
    Density0: 1026.0
```
However, it seems to me that using the reference ocean density in this term is related to the Boussinesq approximation. We might want to change this.